packages/site/src/components/ComponentLinks.tsx

```diff
@@ -21,11 +21,13 @@ export const ComponentLinks = ({
         <Heading level={2}>Links</Heading>
         <Box>
           {links?.map((link, index) => (
-            <Box key={index}>
-              <Link key={index} url={link.url}>
-                {link.label}
-              </Link>
-            </Box>
+            <div key={index} data-storybook-link>
```

Comment:
I think there might be a better way to do this, I was having issues initially selecting the link text but didn't circle back to this, I think we could drop a ticket to get this out in the future (maybe when we get the design pages in).